pyproject.toml

```diff
@@ -210,6 +210,7 @@ ignore_missing_imports = true
 disallow_any_generics = true
 pretty = true
 show_error_context = true
+check_untyped_defs = true
```

Comment:
If we want to add this to ignore-revs then the commit must not contain any manual changes. So this PR must only contain the type ignores, and a follow-up PR should add this config and the ignore-revs item containing the commit hash of this PR.

Reply:
Ah ok. Thanks. It seems this is a repeat of https://github.com/pyvista/pyvista/pull/6664#issuecomment-2364590326: the squash-merge model is not suited for this. 

Our CI will fail if we remove this manual change, so it's not possible to separate the manual change from the automatic changes in this case. So, we won't add this to the blame ignore. Thanks for clarifying this (again).

Reply:
Yup. But why will CI fail? I thought adding _ignores_ would be safe.

Reply:
Ah ok maybe it's good to remove this manual line after all and add it separately. 

Mypy complains with unused ignores. I was somehow thinking it would complain about the new ignores without the rule. But I think you're right it should be OK to remove it here.

Reply:
Unfortunately it is as I thought: the new ignores are considered unused without the rule and CI is failing if we remove it: https://github.com/pyvista/pyvista/actions/runs/11839504031/job/32991100716#step:5:3818

Reply:
Bummer. Thanks for trying.